### ISSUE TITLE
The Feedback Tab does not work

### ISSUE THREAD

**Issue by JamesVanWaza** [CodePilotai/codepilot#5]
I tried to submit feedback, but nothing happens, willing to submit a video. 

Also is it possible for you to update Google Chrome, every time, it says reinstall a newer version.

**Comment by petebytes** [comment]
Sorry for the delay in responding - I was about to update electron/chrome but have some broken unit tests, so held off.

Regarding the feedback - feel free to submit that as a issue here on github.  I will change the submit interaction within the codepilot app as soon as some time frees up.

**Comment by JamesVanWaza** [comment]
@petebytes let me know when you update the Chrome.

**Comment by JamesVanWaza** [comment]
@petebytes did you update the Chrome Version?

**Comment by petebytes** [comment]
Tried yesterday, but the upgrade broke the App :(
I created a new branch https://github.com/CodePilotai/codepilot/tree/petebytes/update-dependencies if you want to take a look.

**Comment by JamesVanWaza** [comment]
@petebytes still working on the branch - dependency, I updated the node_modules on package.json. 

Also do you know how to resolve this error below, see screenshot. 

<img width="1342" alt="Screen Shot 2019-08-05 at 6 31 38 PM" src="https://user-images.githubusercontent.com/4000076/62499152-9e081380-b7af-11e9-9fda-c5163a1fd756.png">

**Comment by JamesVanWaza** [comment]
> @petebytes still working on the branch - dependency, I updated the node_modules on package.json.
> 
> Also do you know how to resolve this error below, see screenshot.
> 
> <img alt="Screen Shot 2019-08-05 at 6 31 38 PM" width="1342" src="https://user-images.githubusercontent.com/4000076/62499152-9e081380-b7af-11e9-9fda-c5163a1fd756.png">

**Comment by cameronadams777** [comment]
@JamesVanWaza Just to try and track down the issue that you are seeing, it appears that you do not have `vue-cli-service` installed. This generally comes with `vue-cli` 3. That being said, while attempting to update the versions in package.json, did you happen to remove `node_modules` directory? Maybe try and run `yarn` one more time and see if that resolves the issue.

**Comment by JamesVanWaza** [comment]
@cameronadams777 I believe I was able to get it open. is this the correct screenshot?

Also, multiple dependencies need to be updated. 

<img width="1086" alt="Screen Shot 2020-05-04 at 10 58 24 AM" src="https://user-images.githubusercontent.com/4000076/80980186-570a8580-8df6-11ea-9789-b5b5337bb8d5.png">
<img width="1680" alt="Screen Shot 2020-05-04 at 11 00 04 AM" src="https://user-images.githubusercontent.com/4000076/80980271-6f7aa000-8df6-11ea-9134-8caec5d373d2.png">

**Comment by cameronadams777** [comment]
@JamesVanWaza Yes I agree. I have just bumped the versions that needed to be updated to get the app working again. If you want to take on updating the dependencies, feel free.

**Comment by JamesVanWaza** [comment]
Ok I will fork and do the updates then let you know

**Comment by JamesVanWaza** [comment]
@cameronadams777 Review pull request
